packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -325,7 +325,7 @@ exports[`.toBe() fails for: [] and [] 1`] = `
 <dim>If it should pass with deep equality, replace \\"toBe\\" with \\"toStrictEqual\\"</>
 
 Expected: <green>[]</>
-Received value has no visual difference"
+Received: has same serialization"
```

Comment:
"serialises to the same string"?

Reply:
Yes, let’s prefer a verb to a noun. What do you think about the following:

```sh
Expected: []
Received: different instance that serializes to the same string
```

And this reminds me to think about a sentence for `toEqual` and `toStrictEqual` in `ExpectAPI.md` to explain that different functions are not deep equal, see #8166

Reply:
I decided on the shorter statement suggested by Simen.

The called function cannot assume that the longer statement is true.